SignNow.Net/Interface/IOAuth2Service.cs

```diff
@@ -0,0 +1,16 @@
+using SignNow.Net.Model;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SignNow.Net.Interface
+{
+    public interface IOAuth2Service
+    {
+        Task<Token> GetTokenAsync(string login, string password, Scope scope, CancellationToken cancellationToken = default(CancellationToken));
+        Task<Token> GetTokenAsync(string code, Scope scope, CancellationToken cancellationToken = default(CancellationToken));//TODO: https://pdffiller.atlassian.net/wiki/spaces/API2/pages/911769793/Grant+type+authorization+code
```

Comment:
Pls remove comment. You can configure VS to exclude such things in commit

Reply:
@AlexNDRmac why? This is hint for developer where to find information on implementation of the method.